lcli/Cargo.toml

```diff
@@ -45,3 +43,9 @@ tracing-subscriber = { workspace = true }
 tree_hash = { workspace = true }
 types = { workspace = true }
 validator_dir = { workspace = true }
+
+[target.'cfg(not(target_os = "windows"))'.dependencies]
+malloc_utils = { workspace = true, features = ["jemalloc"] }
+
+[target.'cfg(target_os = "windows")'.dependencies]
+malloc_utils = { workspace = true, features = [] }
```

Comment:
These changes ensure that `lcli` actually uses jemalloc on Linux, which is potentially relevant for the benchmarking subcommands.